app/api/stripe/subscription/search/route.ts

```diff
@@ -0,0 +1,29 @@
+import stripeClient from "@/lib/stripe/client";
+import { NextRequest } from "next/server";
+
+export async function GET(req: NextRequest) {
+  const accountId = req.nextUrl.searchParams.get("accountId");
```

Comment:
Is the goal of this API to get active subscriptions for an account?
- if yes, why not lookup subscription for the `email_address` instead of `accountId`?
- `email` can be directly queried from the stripe API so your query can be more `specific` 🎯